public/app.js

```diff
@@ -0,0 +1,347 @@
+const APIController = (function() {
+
+  const clientId = 'e91988fcc1674aa88374f24be4844f85';
+  const clientSecret = '10badbb697f544ecb15e1efd7659a0b0';
+
+  const send = async (url,params) => {
+      let token = localStorage.my_token;
+      if (token == null){
```

Comment:
стоит сравнивать значения без приведения типов.
```suggestion
      if (token === null){
```